### ISSUE TITLE
Redstone causes rendering issues with multicore chunkloading

### ISSUE THREAD

**Issue by facetorched** [Rexhunter99/TFC-Basic-Shaders#6]
Using this shader with multicore chunkloading. everything is fine until I make a simple torch burnout clock. The clock lags visually and so you can't tell if redstone components are on or off. The presence of the redstone clock also causes issues with rendering other things: when you break blocks they don't disappear until the player moves.

destroying the redstone clock or turning off multicore chunkloading immediately fixes the issue

**Comment by Rexhunter99** [comment]
This is an issue with Optifine for 1.7.10 and not with my shaders, it also occurs with dynamic lighting.
I might be able to whip up a hacky fix that handles the lighting in the shader instead of letting Optifine do it but I'm not sure if it will apply to redstone (as that's a placed world item and the shader's just lack too much information to be able to realistically determine that stuff.

**Comment by facetorched** [comment]
Yeah using this shader didn't cause the issue. Nuck asked me to open a ticket since this shader aims to fix these kinds of issues :)

**Comment by nuckable** [comment]
Would it be possible to override the code for redstone to properly forward the necessary information?
